ruby/vampires.rb

```diff
@@ -0,0 +1,53 @@
+puts "How many employees are you processing today?"
+  x = gets.chomp.to_i
+
+while x > 0
+
+  puts "What is your name?"
+  name = gets.chomp
+  puts "How old are you?"
+  age = gets.chomp.to_i
+  puts "What year were you born in?"
+  year = gets.chomp.to_i
+  puts "Our company cafeteria orders garlic bread...should we order some for you? (yes or no)"
+  garlic_bread = gets.chomp
+  puts "Would you like to enroll in our company's health insurance?"
+  health_insurance = gets.chomp
+
+age_match = 2017 - year
```

Comment:
Consider checking  the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.